tests/core/test_path_package_functions.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+from re import escape
+
+from pytest import fixture, mark, raises
+
+from sprout.core import (
+    path_package,
+    path_package_database,
+    path_package_properties,
+    path_packages,
+)
+from tests.core.directory_structure_setup import (
+    create_test_package_structure,
+)
+
+
+# Given one package
+@fixture
+def tmp_sprout_root(monkeypatch, tmp_path):
+    """Set up test package folder structure and return temp root directory"""
+    SPROUT_ROOT = str(tmp_path)
+    monkeypatch.setenv("SPROUT_ROOT", SPROUT_ROOT)
```

Comment:
```suggestion
    monkeypatch.setenv("SPROUT_ROOT", str(tmp_path))
```
If you convert the path to string in here then you can return it as a `Path` and then you don't have to reconvert it to `Path` in the tests. Same in other test file.

Reply:
Omg, ofc! Thanks 😁